src/main/resources/db/migration/postgres/V1_333_3__insert_security_groups.sql

```diff
@@ -0,0 +1,63 @@
+DO $$
+DECLARE
+	recordToBeMigrated record;
+	newId integer;
+	oldId integer;
+	maxId integer;
+BEGIN
+perform setval('grp_seq', 1000, false);
+RAISE NOTICE 'migrating duplicate records';
+FOR recordToBeMigrated IN
+    SELECT sg1.grp_id newId, sg2.group_name, sg2.grp_id oldId FROM security_group sg1, security_group sg2
```

Comment:
Add a comment here as to what we are doing. Not clear that old is greater than 1000 and new is less than 1000

Reply:
Maybe worth assigning 1000 a variable name and using that

Reply:
comment and constant added.